kogito-usertasks-custom-lifecycle-quarkus/README.md

```diff
@@ -13,9 +13,32 @@ This example shows
 * working with user tasks
 * four eye principle with user tasks
 	
-	
+* Diagram	
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
+	
+Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties.png"></p>
+
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties2.png"></p>
+
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties3.png"></p>
+
+* First Line Approval
+<p align="center"><img src="docs/images/firstLineApprovalUserTask.png"></p>
+
+* First Line Approval
+<p align="center"><img src="docs/images/firstLineApprovalUserTask2.png"></p>
 
+* First Line Approval
+<p align="center"><img src="docs/images/firstLineApprovalUserTaskAssignments.png"></p>
+
+* Second Line Approval
+<p align="center"><img src="docs/images/secondLineApprovalUserTask.png"></p>
+
+* Second Line Approval
+<p align="center"><img src="docs/images/secondLineApprovalUserTaskAssignments.png"></p>
 
 In addition this quickstart also shows how to use custom life cycle for user tasks that uses custom life cycle phases
 compared to those supported by default.
```

Comment:
(line 95) swagger-ui is only accessible in dev mode (so with mvn clean package quarkus:dev)
See https://quarkus.io/guides/openapi-swaggerui#use-swagger-ui-for-development

Reply:
Done @radtriste